### PR TITLE
included more content in html, new css styles, javascript for tabs an…

### PR DESCRIPTION
**Pull request by dexdreamin**
…d slideshows and included a new font

### CODE REVIEW

**Comment by dexdreamin** [comment]
> …d slideshows and included a new font

File: chatbot.html [hunk 1]
```diff
@@ -358,39 +358,319 @@ <h2>DIETBOT</h2>
...
+        font-size: 20px;
         
     }
     .content-background{
         background-color: silver;
-        opacity: 0.8;
+        opacity: 0.9;
         padding:20px
     }
+
```
Comment (dexdreamin) [inline chatbot.html:389]: Added styling for the tabs and slides, using old brown paper design to match the theme. 
For tab content, there is a styled description using the old brown paper theme as well.

File: chatbot.html [hunk 2]
```diff
@@ -358,39 +358,319 @@ <h2>DIETBOT</h2>
...
+        <div id="avoid" class="tabcontent" style="background:url('website_images/food-waste.jpeg') no-repeat center center fixed; background-size: 100%;">
+          <div class="tab-desc">
+            <h2>6. Avoid going for too much food</h2>
+            <p>Lastly, planning of what amount of food is enough and sufficient for you in the long run.</p>
+            <p>Before spending on buying more food, think first if it is neccessary to buy first and whether it is enough to suit your needs. <br>
+              If it is in your plan that you do not need such a large amount of food, then you can reduce the portion of what you buy. Always make a conscious effort to serve food in smaller portions. <br>
+              Even in restaurants and if you do happen to have more than what you can finish on the table, there is never a problem to take the food back as a take-away to finish it back home.</p>
+          </div>
+        </div>
+
```
Comment (dexdreamin) [inline chatbot.html:604]: Above here, I simplified the content page to be more reader-friendly for our users and can select different tabs to find out more information of what they can do as an individual. With that, it reduces the "wordiness" of the content and we can direct the focus of our users to what information they want to find out more about.

File: chatbot.html [hunk 3]
```diff
@@ -358,39 +358,319 @@ <h2>DIETBOT</h2>
...
+        tablinks[i].className = tablinks[i].className.replace(" active", "");
+      }
+      
+      // Show the current tab, and add an "active" class to the button that opened the tab
+      document.querySelector(id_name).style.display = "block";
+      evt.currentTarget.className += " active";
+
+
+    }
+    </script>
```
Comment (dexdreamin) [inline chatbot.html:656]: Above here, it is my JavaScript program that maintains all the tab functions and its overall interactivity to the users.

File: chatbot.html [hunk 4]
```diff
@@ -358,39 +358,319 @@ <h2>DIETBOT</h2>
...
+          dots[i].className = dots[i].className.replace(" active", "");
+        }
+        slides[slideIndex-1].style.display = "block";  
+        dots[slideIndex-1].className += " active";
+        setTimeout(showSlides, 3000); // Change image every 3 seconds
+
+        
+       
+      }
+    </script>
```
Comment (dexdreamin) [inline chatbot.html:684]: Lastly, here is a JavaScript program that maintains the function of the carousel slides switching from one page to another automatically.

**Comment by iamsven2005** [comment]
lgtm

**Pull request merged**